pkg/shell/hosts_dialog.jsx

```diff
@@ -69,6 +69,14 @@ function is_method_supported(methods, method) {
     return result ? result !== "no-server-support" : false;
 }
 
+function handle_submit(callback) {
+    return event => {
+        callback();
+        event.preventDefault();
+        return false;
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19018-20230626-125250-b763fa4b-fedora-38-devel-networking//Coverage/lcov/github-pr.diff.gcov.html)